vulkano/src/memory/allocator/mod.rs

```diff
@@ -396,6 +398,8 @@ impl Debug for dyn MemoryAllocator {
 /// .unwrap();
 /// ```
 ///
+/// #### Sequential writes from host, indirect device access
+///
 /// For staging, the resource is only ever written to sequentially. Also, since the device will
 /// only read the staging resourse once, it would yield no benefit to place it in device-local
```

Comment:
```suggestion
/// only read the staging resource once, it would yield no benefit to place it in device-local
```

Typo that was there before, might as well fix it now.